iidm/iidm-impl/src/main/java/com/powsybl/iidm/network/impl/DanglingLineBoundaryImpl.java

```diff
@@ -53,6 +51,21 @@ public double getQ() {
         return new SV(t.getP(), t.getQ(), getV(b), getAngle(b)).otherSideQ(parent);
     }
 
+    @Override
+    public Branch.Side getSide() {
```

Comment:
Same question.

Reply:
For dangling line, the side is always null though...